test/unit/tools/Stubs.scala

```diff
@@ -64,6 +64,7 @@ trait Stubs {
       |metrics.showSamples=false
       |metrics.jvm=false
       |metrics.logback=false
+      |draft.timeToLive=1d
```

Comment:
What;s this change for?

Reply:
To render the summary page for a draft we need that adding into the config apparently,
otherwise it breaks